supporting_content/metadata/schema_creation_scenarios.md

```diff
@@ -0,0 +1,13 @@
+# Scheme Creation Scenarios
+
+## The following is a list of scenarios and their schema update requirements
+
+
+| Scenario                                                                                                 | Requires New Schema object	 | Requires Change in Provider Version | 	Requires Change Focus Version |
```

Comment:
This whole section needs alignment on capitalization -- here's just the first line scrubbed.

```suggestion
| Scenario                                                                                                 | Requires New Schema object	 | Requires Change in Provider Version | 	Requires Change FOCUS Version |
```